mm2src/coins/utxo/utxo_tests.rs

```diff
@@ -601,3 +611,13 @@ fn get_tx_details_coinbase_transaction() {
 
     block_on(fut);
 }
+
+#[test]
+#[ignore]
+fn test_electrum_rpc_client_error() {
+    let client = electrum_client_for_test(&["electrum1.cipig.net:10060", "electrum2.cipig.net:10060"]);
+
+    let empty_hash = H256Json::default();
+    let err = unwrap_err!(client.get_verbose_transaction(empty_hash).wait());
+    log!((err));
```

Comment:
Could you please remove `#[ignore]` and assert that error msg contains address of electrum server?

Reply:
Unfortunately I couldn't use neither json! macro nor JsonRpcError serde_json deserialization,
instead of these I added comparing of static expected string and formatted actual string

Reply:
I think it could be sufficient and a bit simpler to just `assert!(error_msg.contains("electrum1.cipig.net:10060"))`, you may use such approach next time, any option is ok if works.